### ISSUE TITLE
[Build Error]: Error while building ng-packagr locally

### ISSUE THREAD

**Issue by Sibiraj-S** [dherges/ng-packagr#670]
## Type of Issue

```
[x] Bug Report
[ ] Feature Request
```

## Description

```
> rimraf dist && yarn schema

yarn run v1.5.1
$ json2ts --input src/ng-package.schema.json --output src/ng-package.schema.ts
Done in 1.08s.

> ng-packagr@2.2.0 build E:\Github\ng-packagr
> tsc -p src/tsconfig.packagr.json

src/lib/flatten/rollup-tsc.ts(24,3): error TS2322: Type '{ code: string; map: string; }' is not assignable to type 'SourceDescription'.
  Types of property 'map' are incompatible.
    Type 'string' is not assignable to type 'RawSourceMap'.
```

#### How To Reproduce

I use windows 10 PC, Just clone `ng-packagr` and run `npm run build` locally

I guess the build runs successfully in circle-ci.

**Comment by zelibobla** [comment]
Any explanation of why is this closed? Facing the same issue 
```
src/lib/flatten/rollup-tsc.ts(25,3): error TS2322: Type '{ code: string; map: string; }' is not assignable to type 'SourceDescription'.
  Types of property 'map' are incompatible.
    Type 'string' is not assignable to type 'RawSourceMap'.

```

Darwin 17.4.0
node v7.7.2
npm  v4.1.2
ng-packagr@2.4.2

**Comment by dherges** [comment]
Hi @zelibobla @Sibiraj-S, it's a weird dependency issue. Should be solved by #769
